src/app/Dashboard/LayoutTemplateGroup.tsx

```diff
@@ -124,49 +124,48 @@ export const LayoutTemplateGroup: React.FC<LayoutTemplateGroupProps> = ({
         {props.templates.map((template) => {
           const level = smallestFeatureLevel(template.cards);
           return (
-            <FeatureFlag key={template.name} level={level}>
-              <div
-                ref={scrollRef}
-                className={
-                  selectedTemplate?.name === template.name && selectedTemplate.vendor == template.vendor
-                    ? 'layout-template-card__featured'
-                    : undefined
-                }
+            <div
+              key={template.name}
+              ref={scrollRef}
+              className={
+                selectedTemplate?.name === template.name && selectedTemplate.vendor == template.vendor
+                  ? 'layout-template-card__featured'
+                  : undefined
+              }
+            >
+              <CatalogTile
+                featured={selectedTemplate?.name === template.name && selectedTemplate.vendor == template.vendor}
+                id={template.name}
+                key={template.name}
+                icon={iconify(template.vendor)}
+                title={template.name}
+                vendor={template.vendor}
+                onClick={() => handleTemplateSelect(template)}
+                badges={[
+                  level !== FeatureLevel.PRODUCTION && (
+                    <FeatureFlag key={template.name + '-feature-flag'} level={level}>
```

Comment:
I think we can remove this. Either way it will show with or without FeatureLevel here.